src/main/java/org/jenkinsci/plugins/workflow/steps/AbstractSynchronousNonBlockingStepExecution.java

```diff
@@ -74,7 +75,7 @@ public void onResume() {
         getContext().onFailure(new Exception("Resume after a restart not supported for non-blocking synchronous steps"));
     }
 
-    @Override public String toString() {
+    @Override public @Nonnull String getStatus() {
```

Comment:
So... we're overriding a method with CheckForNull with one that is Nonnull?  I take it that findbugs is smart enough to handle this case correctly?